handler/src/test/java/io/netty/handler/ipfilter/IpSubnetFilterTest.java

```diff
@@ -152,6 +152,7 @@ public void testBinarySearch() {
         ipSubnetFilterRuleList.add(buildRejectIP("200.200.200.200", 32));
         ipSubnetFilterRuleList.add(buildRejectIP("108.0.0.0", 4));
         ipSubnetFilterRuleList.add(buildRejectIP("10.10.10.10", 8));
+        ipSubnetFilterRuleList.add(buildRejectIP("2001:db8:abcd:0000::", 52));
```

Comment:
Mixed version IPs should be in a separate test, as otherwise _this_ test will now fail because of the new exception thrown from the constructor, right?

Reply:
Forgot to remove :p